templates/public-adventures.html

```diff
@@ -0,0 +1,62 @@
+
+{% extends "auth.html" %}
+
+{% block regular_content %}
+
+    <div id="filters"
+        class="flex flex-col sm:flex-row items-center gap-2
+        justify-around bg-blue-400 shadow-md rounded-lg p-4 mb-4"
+        _="on load set $filtered to []">
+        <div>{{_('apply_filters')}}</div>
+        <div id="search" class="flex-1">
+            <input id="search_adventure"
+                type="search"
+                placeholder="{{_('search')}}"
+                class="border text-black border-gray-500 rounded h-8 px-2 w-full"
+                _="on keyup debounced at 500ms set searchTerm to my value 
+                    hedyApp.applyFilter(searchTerm, 'search', $filtered)">
+        </div>
+        <div class="flex gap-2">
+            <div id="languages">
+                <select id="language"
+                    name="language"
+                    required
+                    class="block appearance-none w-full border border-gray-200 text-gray-700 p-2 rounded"
+                    _="on change set lang to my value
+                        hedyApp.applyFilter(lang, 'lang', $filtered)">
+                    <option value="">{{_('select_lang')}}</option>
+                    {% for lang in available_languages %}
+                        <option value="{{lang or current_language().lang}}">{{lang_to_sym(lang or current_language().lang)}}</option>
+                    {% endfor %}
+                </select>
+            </div>
+            <div id="tags">
+                <select id="tag"
+                    name="tag"
+                    required
+                    class="block appearance-none w-full border border-gray-200 text-gray-700 p-2 rounded"
+                    _="on change set tag to my value
+                        hedyApp.applyFilter(tag, 'tag', $filtered)">
+                    <option value="">{{_('select_tag')}}</option>
+                    {% for tag in available_tags %}
+                        <option value="{{tag}}">{{tag}}</option>
+                    {% endfor %}
+                </select>
+            </div>
+        </div>
+    </div>
+    <div id="adventures" class="card-container gap-2">
+        {% if adventures|length == 0 %}
+            <div class="grid p-5 text-center bg-white shadow-md rounded-lg transition-shadow hover:shadow-lg outline-blue-400 hover:outline hover:outline-2">
+                {{_('no_adventures_yet')}}
+                <button class="green-btn mt-4" id="create_adventure_button"
+                    onclick='hedyApp.create_adventure({{_('adventure_prompt')|default(None)|tojson}})'
+                    {% if second_teacher and role=='viewer' %}disabled{% endif %}>{{_('create_adventure')}}</button>
+            </div>
+        {%endif%}
+        {% for adventure in adventures %}
+            {{ render_partial('htmx-adventure-card.html', user=user, adventure=adventure) }}
+        {% endfor %}
+    </div>
```

Comment:
Maybe the same can be said about these cards? I like them! But might be worth it to add a UI similar to the one we use for programs? You can find that code here: https://github.com/hedyorg/hedy/blob/pub-adventures/templates/macros/public-programs.html

Reply:
I don't like tables and there there's tables. In the public adventures view, we wanna make new design, enhance and generalize it incrementally.

Reply:
Yes, I understand. But the card themselves, which is the file I linked, are just a div.

Reply:
I don't really understand, but we will surely make changes on this page moving forward, so we can discuss it more later ;)